Content.Server/Xenoarchaeology/Equipment/Systems/NodeScannerSystem.cs

```diff
@@ -15,6 +16,29 @@ public sealed class NodeScannerSystem : EntitySystem
     public override void Initialize()
     {
         SubscribeLocalEvent<NodeScannerComponent, AfterInteractEvent>(OnAfterInteract);
+        SubscribeLocalEvent<NodeScannerComponent, GetVerbsEvent<UtilityVerb>>(AddScanVerb);
+    }
+
+    private void AddScanVerb(EntityUid uid, NodeScannerComponent component, GetVerbsEvent<UtilityVerb> args)
+    {
+        if (!args.CanAccess)
+            return;
+
+        if (!TryComp<ArtifactComponent>(args.Target, out var artifact) || artifact.CurrentNodeId == null)
+            return;
+
+            var verb = new UtilityVerb()
+            {
+                Act = () =>
+                {
+                    CreatePopup(uid,args.Target,artifact);
+                },
+                Text = Loc.GetString("node-scan-tooltip")
+            };
+
+            args.Verbs.Add(verb);
```

Comment:
Needs to be formatted correctly. Unindent this block and add spaces after commas (,).